server/main.lua

```diff
@@ -346,7 +346,7 @@ QBCore.Functions.CreateCallback('mdt:server:GetProfileData', function(source, cb
 		pp = ProfPic(target.charinfo.gender),
 		licences = licencesdata,
 		dob = target.charinfo.birthdate,
-		fingerprint = target.metadata.fingerprint,
+		fingerprint = '',

```

Comment:
Couldn't this just be
```suggestion
		fingerprint = target.metadata.fingerprint or '',

```
So the '' is a fallback if the first value returns nil?

Reply:
no because then it gets added automatically again, its the same for the mdtinfo

Reply:
Oh, I put it in the wrong context. Woops.